runtime/metering.go

```diff
@@ -0,0 +1,59 @@
+package runtime
+
+type MetringOperationType uint
+
+const (
+	// base [0-10)
+	OpTypeStatement MetringOperationType = iota
```

Comment:
Prefix with the type name, e.g. 

```suggestion
	ComputationOperationStatement ComputationOperation = iota
```

Reply:
renamed all to match how memoryKinds are defined